sdk/src/driftClient.ts

```diff
@@ -2233,8 +2255,9 @@ export class DriftClient {
 
 		const tx = await this.buildTransaction(withdrawIxs, {
 			...(txParams ?? this.txParams),
-			computeUnits: 1_400_000,
+			computeUnits: 1_400_000, // TODO : Is this still necessary?
```

Comment:
this was when a user had a tx running outta cu's for withdraw, if we use simulation we dont need to use max